method/SCAN-IT/scanit.yml

```diff
@@ -24,5 +24,5 @@ dependencies:
   - networkx=2.6.3
   - leidenalg=0.10.1
   - pip:
-      - scanit==0.1
       - somoclu==1.7.6
+      - "--editable=git+https://github.com/zcang/SCAN-IT.git#egg=scanit"
```

Comment:
I don't think this should be an editable install. Also please add a specific copmmit hash to be installed as unfortunately there are now release tags available